app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -334,6 +334,15 @@ def parse_persistent_volume(persistent_volume)
       new_result
     end
 
+    def parse_capacity(hash)
```

Comment:
this method modifies its input, so I would either change it to `parse_capacity!` - but bang methods should be avoided imho. So. I'd make it return a new hash. And shouldnt it be private? 
Actually I liked the inlined version better :)
How about setting it up right before the merge:
```ruby
capacity = persistent_volume.spec.capacity.to_h.transform_values do |iec_number|
  PersistentVolume.parse_iec_number(iec_number)
end
new_result.merge!(
 :capacity       => capacity
)